client/app/templates/collection/browse.hbs

```diff
@@ -0,0 +1 @@
+{{browse-list model=model changeRoute=(action 'changeRoute')}}
```

Comment:
Can the `changeRoute=(action 'changeRoute')` be removed here as well @caneruguz?